src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -5404,6 +5385,19 @@ public static final Modifiers recalculateAdjustments(
           Modifiers.ITEMDROP,
           newModifiers.getExtra(Modifiers.ITEMDROP),
           "Item:[" + ItemPool.BROKEN_CHAMPAGNE + "]");
+      // Subtract fightMods back out.
+      if (fightMods != null) {
```

Comment:
what are `fightMods`, and why are they being subtracted here?

Reply:
`fightMods` comes from the lookup string `Generated:fightMods`. It stores modifiers that can occur during a fight for the duration of the fight, a prime example being Otoscope. Squint doubles Otoscope, but Champagne doesn't, and we should reproduce that behavior. The cleanest way I found to do it was to have it counted on extras and then subtract it back out later.

Reply:
Alternatively, I could rewrite it to add fightMods in explicitly at the Squint stage. That might be better.

Reply:
it does seem clearer that way